tests/feature/karmalist/feature_test.go

```diff
@@ -0,0 +1,16 @@
+package karmalist
+
+import (
+	"testing"
+
+	"github.com/jakevoytko/crbot/testutil"
+)
+
+func TestKarmaList_NoResponse(t *testing.T) {
+	runner := testutil.NewRunner(t)
+
+	// Add Karma using builtin
+	runner.SendMessageIgnoringResponse(testutil.MainChannelID, "?++ Testing")
+	// Test Karmalist verifying builtin success by proxy
+	runner.SendKarmaListMessage(testutil.MainChannelID)
```

Comment:
Can you also call this before running `?++ testing` ?

Reply:
I'm not sure what you mean here? Wouldn't this be moot given the modelhelper strategy you detailed below?